src/System.Net.Mail/tests/Unit/SmtpDateTimeTest.cs

```diff
@@ -76,17 +76,11 @@ public void SmtpDateTime_WithDateThatHasKindSetToLocal_ShouldSetTimeZoneCorrectl
         }
 
         [Theory]
-        [InlineData(ValidCompleteDateString, 2009, 5, 17, 15, 34, 7, "GMT", DateTimeKind.Unspecified)]
-        [InlineData(ValidDateStringWithKnownShortHandTimeZone, 2009, 5, 17, 15, 34, 7, "GMT", DateTimeKind.Unspecified)]
-        [InlineData(ValidDateStringWithNoDayOfWeek, 2009, 5, 17, 15, 34, 7, "GMT", DateTimeKind.Unspecified)]
-        [InlineData(ValidDateStringWithOnlyTabsAsWhitespace, 2009, 5, 17, 15, 34, 7, "GMT", DateTimeKind.Unspecified)]
-        [InlineData(ValidDateStringWithTrailingWhitespaceAndCommentAfterTimeZone, 2009, 5, 17, 15, 34, 7, "GMT", DateTimeKind.Unspecified)]
-        [InlineData(ValidDateStringWithMixedTabsAndSpacesAsWhitespace, 2009, 5, 17, 15, 34, 7, "GMT", DateTimeKind.Unspecified)]
+        [InlineData(2009, 5, 17, 15, 34, 7, "GMT")]
         public void SmtpDateTime_CreatedFromDateTimeString_ShouldParseCorrectly(
-            string input,
```

Comment:
Shouldn't the input remain here and be `ValidDateStringWithKnownShortHandTimeZone` which result is used further down for in the assert equal checks?

Reply:
Yup, it should have been used in the ParseValue call instead of the hardcoded string.  Fixed.